data/src/main/java/org/oppia/android/data/backends/gae/model/GaeFeedbackReportingEntryPoint.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.android.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Data class for feedback reporting entry point represented in the backend domain model
+ * TODO(#2801): Link backend domain model
```

Comment:
Nit: TODOs shouldn't be in KDocs since they're not actually part of the class's documentation.

Reply:
Done